continuous_integration/environment-mindeps-array.yaml

```
@@ -6,6 +6,7 @@ dependencies:
   - packaging=20.0
   - python=3.8
   - pyyaml>=5.3.1
+  - click>=6.6
```

Let's pin to the earliest supported version in the `mindeps` builds

```suggestion
  - click=6.6
```